common/achievements.cpp

```diff
@@ -89,9 +89,10 @@ bool AchievementsManager::setAchievement(const String &id, const String &display
 
 	if (!displayedMessage.empty() && g_system) {
 		U32String msg;
-		msg = Common::U32String::format("%S\n%S",
-			_("Achievement unlocked!").c_str(),
-			Common::U32String(displayedMessage).c_str()
+		msg = Common::U32String::format(
+		    "%S\n%S",
+		    _("Achievement unlocked!").c_str(),
+		    displayedMessage.decode(Common::kLatin1).c_str()
```

Comment:
I don't know it's used only by testbed (ASCII) and wintermute. For later I have no idea which encoding it uses

Reply:
I think generally speaking wintermute uses unicode? so my gut feeling is that UTF-8 might make more sense here. But it would be better to check with @lolbot-iichan.

Or maybe the displayedMessage should be passed as a `Common::U32String` so that we don't have to worry about the encoding and this is the responsibility of the code using the `AchievementsManager`.